README.md

```diff
@@ -246,6 +246,22 @@ req.checkBody({
 });
 ```
 
+You can also define location against to validate specific schema validator by adding `in` parameter as fallowing:
+
+```javascript
+req.checkBody({
```

Comment:
This example makes the new feature seem confusing and counterintuitive. I would recommend giving you main example with `check()` and then have a note saying that if you happen to use `checkBody/Query/Params`, the `in` attribute will take precedence.